cluster/operator-install.sh

```diff
@@ -19,4 +19,5 @@ set -ex
 export VERSION=v0.0.0
 
 ./cluster/kubectl.sh create -f _out/vm-import-operator/${VERSION}/operator.yaml
+./cluster/kubectl.sh create -f _out/vm-import-operator/${VERSION}/vmimportconfig_cr.yaml
 ./cluster/kubectl.sh wait deploy/vm-import-operator -n kubevirt --for=condition=Available --timeout=600s
```

Comment:
i think the correct condition will be waiting for deploy/vm-import-controller to be ready before proceeding with tests